### PR TITLE
Tweak footer.php

### PR DESCRIPTION
**Pull request by XhmikosR**
**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

I tested it as much as I could, but the more eyes the better 🙂

vDev:

1-vdev

---

master with updates:

2-updates

Generated HTML beautified for demonstration purposes:

```html
<footer class="main-footer">
  <div class="row row-centered text-center">
    <div class="col-xs-12 col-sm-6">
      <strong><a href="https://pi-hole.net/donate/" rel="noopener" target="_blank"><i class="fa fa-heart text-red"></i> Donate</a></strong> if you found this useful.
    </div>
  </div>

  <div class="row row-centered text-center version-info">
    <div class="col-xs-12 col-sm-8 col-md-6">
      <ul class="list-inline">
        <li>
          <strong>Pi-hole Version</strong>
          <a href="https://github.com/pi-hole/pi-hole/releases/v5.0" rel="noopener" target="_blank">v5.0</a>
           · <a href="https://github.com/pi-hole/pi-hole/releases/latest" rel="noopener" target="_blank" class="alert-link lookatme">Update available!</a>
        </li>
        <li>
          <strong>Web Interface Version</strong>
          <a href="https://github.com/pi-hole/AdminLTE/releases/v5.0" rel="noopener" target="_blank">v5.0</a>
           · <a class="lookatme" href="https://github.com/pi-hole/AdminLTE/releases/latest" rel="noopener" target="_blank">Update available!</a>
        </li>
        <li>
          <strong>FTL Version</strong>
          <a href="https://github.com/pi-hole/FTL/releases/v5.0" rel="noopener" target="_blank">v5.0</a>
           · <a class="lookatme" href="https://github.com/pi-hole/FTL/releases/latest" rel="noopener" target="_blank">Update available!</a>
        </li>
      </ul>
    </div>
  </div>
</footer>
```

We could even link to the devel commits but I thought I'd leave this for later.

### CODE REVIEW

File: scripts/pi-hole/php/footer.php [hunk 1]
```diff
@@ -36,34 +36,69 @@
...
+            <div class="col-xs-12 col-sm-6">
+                <strong><a href="https://pi-hole.net/donate/" rel="noopener" target="_blank"><i class="fa fa-heart text-red"></i> Donate</a></strong> if you found this useful.
+            </div>
+        </div>
+
+        <div class="row row-centered text-center version-info">
+            <div class="col-xs-12 col-sm-8 col-md-6">
+                <?php if (isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?>
+                <ul class="list-unstyled">
+                    <li><strong>Pi-hole Version</strong> <?php echo $coreVersionStr; ?></li>
```
Comment (XhmikosR) [inline scripts/pi-hole/php/footer.php]: We could probably skip the "Version" part?

Comment (PromoFaux) [reply scripts/pi-hole/php/footer.php]: I'm thinking along the lines of: (edited in the browser)

image

@DL6ER / @dschaper - any preference to add here?

Comment (DL6ER) [reply scripts/pi-hole/php/footer.php]: Does this PR make the footer taller? It currently is a one-liner for me
Screenshot from 2020-05-23 10-09-59

Comment (XhmikosR) [reply scripts/pi-hole/php/footer.php]: Yes, it does for non-stable tags.

File: scripts/pi-hole/php/footer.php [hunk 2]
```diff
@@ -36,34 +36,69 @@
...
+<?php
+    // Flushes the system write buffers of PHP. This attempts to push everything we have so far all the way to the client's browser.
+    flush();
+    // Run update checker
+    //  - determines local branch each time,
+    //  - determines local and remote version every 30 minutes
+    require "scripts/pi-hole/php/update_checker.php";
+?>
+
 <?php
```
Comment (DL6ER) [inline scripts/pi-hole/php/footer.php]:
```suggestion

```

File: scripts/pi-hole/php/footer.php (hunk 1 above)
Comment (XhmikosR) [reply scripts/pi-hole/php/footer.php]: So, @DL6ER what's your take about dropping the "Version" part? It seems a little redundant after discussing it with @PromoFaux

Comment (DL6ER) [reply scripts/pi-hole/php/footer.php]: I agree

**Review by DL6ER — APPROVED** [review]
Screenshot at 2020-05-25 20-26-54

**Comment by DL6ER** [comment]
Merging despite of a code checking error as the error is not coming from this PR.

**Comment by yubiuser** [comment]
Don't know where to put it else, but I found this FR and find it useful. As you're updating the footer anyway maybe it is possible to display version in mobile also?
https://discourse.pi-hole.net/t/pi-hole-version-and-web-interface-version-on-a-mobile-device/10305

**Comment by XhmikosR** [comment]
It's already done in this PR

**Pull request merged**